modules/apps/redirect/redirect-test/src/testFunctional/tests/Redirect.testcase

```diff
@@ -357,6 +357,57 @@ definition {
 			sourceURL = "${portalURL}/web/test-site-name/new-source-page");
 	}
 
+	@description = "This ensures that a URL can be ignored after reactivation."
+	@priority = "5"
+	test IgnoreURLAfterReactivation {
+		var portalURL = PropsUtil.get("portal.url");
+
+		Redirect.openRedirectAdmin(siteURLKey = "test-site-name");
+
+		Redirect.addEntry(
+			destinationURL = "${portalURL}/web/test-site-name/test-destination-page1",
+			siteName = "test-site-name",
+			sourceURL = "test-source-page");
+
+		Redirect.openRedirectAdmin(siteURLKey = "test-site-name");
+
+		AssertClick(
+			key_rowEntry = "${portalURL}/web/test-site-name/test-source-page",
+			locator1 = "ContentRow#ENTRY_CONTENT_ENTRY_NAME_LINK",
+			value1 = "${portalURL}/web/test-site-name/test-source-page (Opens New Window)");
+
+		Redirect.openRedirectAdmin(siteURLKey = "test-site-name");
+
+		Redirect.viewURLsNotFoundCP(
+			activeIgnoreURL = "true",
+			ignoreURL = "${portalURL}/web/test-site-name/test-destination-page1",
+			requestNumber = "1");
+
+		Redirect.ignoreEntry(ignoreURL = "${portalURL}/web/test-site-name/test-destination-page1");
+
+		Redirect.openRedirectAdmin(siteURLKey = "test-site-name");
+
+		Redirect.ignoreEntry(
+			ignoreURL = "${portalURL}/web/test-site-name/test-destination-page1",
+			Unignore = "true");
+
+		Redirect.viewURLsDefaultNotFoundCP();
+
+		Click(locator1 = "Dropdown#FILTER_AND_ORDER");
+
+		MenuItem.click(menuItem = "Active URLs");
```

Comment:
We can probably turn this into a macro at some point. I'll push this test in for now

Reply:
I'll fix this in a test improvement ticket later. 

thanks @austinchiang